### PR TITLE
feat(icon): hyper link 3 icon

### PR DESCRIPTION
**Pull request by saghdaey**
### Overview

Added the hyperlink 3 icon

With the (upcoming) gamut modal, we can now use a text button in the Share Sandbox modal from the sandbox designs instead of the Fill Button that the gamut dialog generated. As such, we can also include the link icon

Screen Shot 2021-08-13 at 9 12 51 AM

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/plUAVokatDi5L3zexvlkY8/Build?node-id=2%3A838
- [x] Related to JIRA ticket: EGG-1688
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.8.3-alpha.14dc3a.0<br/>@codecademy/gamut-kit@0.5.27-alpha.14dc3a.0<br/>@codecademy/gamut-labs@19.20.2-alpha.14dc3a.0<br/>@codecademy/gamut@36.1.2-alpha.14dc3a.0<br/>@codecademy/styleguide@43.2.2-alpha.14dc3a.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.8.4-alpha.22b47b.0<br/>@codecademy/gamut-kit@0.5.28-alpha.22b47b.0<br/>@codecademy/gamut-labs@19.20.3-alpha.22b47b.0<br/>@codecademy/gamut@36.1.3-alpha.22b47b.0<br/>@codecademy/styleguide@43.2.3-alpha.22b47b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61169f9b0a8d87157c0f128a--gamut-preview.netlify.app>https://61169f9b0a8d87157c0f128a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61169f9b0a8d87157c0f128a>Deploy Logs</a></p>

**Review by mstykmshy — APPROVED** [review]
LGTM! :)

File: packages/gamut-icons/src/svg/regular/hyperlink-3-icon.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>hyperlink-3</title><path d="M10.458,18.374,7.721,21.11a2.853,2.853,0,0,1-3.942,0l-.892-.891a2.787,2.787,0,0,1,0-3.941l5.8-5.8a2.789,2.789,0,0,1,3.942,0l.893.892A1,1,0,0,0,14.94,9.952l-.893-.892a4.791,4.791,0,0,0-6.771,0l-5.8,5.8a4.787,4.787,0,0,0,0,6.77l.892.891a4.785,4.785,0,0,0,6.771,0l2.736-2.735a1,1,0,1,0-1.414-1.415Z"/><path d="M22.526,2.363l-.892-.892a4.8,4.8,0,0,0-6.77,0l-2.905,2.9a1,1,0,0,0,1.414,1.414l2.9-2.9a2.79,2.79,0,0,1,3.941,0l.893.893a2.786,2.786,0,0,1,0,3.942l-5.8,5.8a2.769,2.769,0,0,1-1.971.817h0a2.766,2.766,0,0,1-1.969-.816,1,1,0,1,0-1.415,1.412,4.751,4.751,0,0,0,3.384,1.4h0a4.752,4.752,0,0,0,3.385-1.4l5.8-5.8a4.786,4.786,0,0,0,0-6.771Z"/></svg>
```
Comment (codecaaron) [inline packages/gamut-icons/src/svg/regular/hyperlink-3-icon.svg:1]: Let's rename this to just `hyperlink-icon` (@songstephen thoughts on naming?) and add the slug as a title here.

**Review by codecaaron — CHANGES_REQUESTED** [review]

File: packages/gamut-icons/src/svg/regular/hyperlink-3-icon.svg (hunk 1 above)
Comment (saghdaey) [reply packages/gamut-icons/src/svg/regular/hyperlink-3-icon.svg]: renamed the file and left the title as "hyperlink-3" to match the streamline slug! is that correct?

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.8.4-alpha.c88cdd.0<br/>@codecademy/gamut-kit@0.5.32-alpha.c88cdd.0<br/>@codecademy/gamut-labs@19.20.7-alpha.c88cdd.0<br/>@codecademy/gamut@36.4.2-alpha.c88cdd.0<br/>@codecademy/styleguide@43.6.2-alpha.c88cdd.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://611a61a84a137e3cd077a75e--gamut-preview.netlify.app>https://611a61a84a137e3cd077a75e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/611a61a84a137e3cd077a75e>Deploy Logs</a></p>

**Review by codecaaron — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.8.5-alpha.7985f2.0<br/>@codecademy/gamut-kit@0.5.33-alpha.7985f2.0<br/>@codecademy/gamut-labs@19.21.1-alpha.7985f2.0<br/>@codecademy/gamut@36.4.3-alpha.7985f2.0<br/>@codecademy/styleguide@43.7.1-alpha.7985f2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://611a8094fed150b8861f906a--gamut-preview.netlify.app>https://611a8094fed150b8861f906a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/611a8094fed150b8861f906a>Deploy Logs</a></p>

**Pull request merged**